### PR TITLE
{lib}[GCCcore/8.3.0] libunwind v1.3.1

### PR DESCRIPTION
**Pull request by furstj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/e0878626933f4213e502d55abb7001c5 for a full test report.

File: easybuild/easyconfigs/l/libunwind/libunwind-1.3.1-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+dependencies = [
+    ('XZ', '5.2.4'),
+]
+
+preconfigopts = 'export LIBS="$LIBS -llzma" && export CFLAGS="$CFLAGS -fuse-ld=bfd" && '
+
+sanity_check_paths = {
+    'files': ["include/libunwind.h", "lib/libunwind.%s" % SHLIB_EXT],
+    'files': ['include/libunwind.h', ('lib/libunwind.%s' % SHLIB_EXT, 'lib64/libunwind.%s' % SHLIB_EXT)],
```
Comment (Micket) [inline easybuild/easyconfigs/l/libunwind/libunwind-1.3.1-GCCcore-8.3.0.eb]: You have a double `'files'`

File: easybuild/easyconfigs/l/libunwind/libunwind-1.3.1-GCCcore-8.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'libunwind'
+version = '1.3.1'
+
+homepage = 'http://www.nongnu.org/libunwind/'
```
Comment (Micket) [inline easybuild/easyconfigs/l/libunwind/libunwind-1.3.1-GCCcore-8.3.0.eb]: we should start using https

File: easybuild/easyconfigs/l/libunwind/libunwind-1.3.1-GCCcore-8.3.0.eb (hunk 1 above)
Comment (furstj) [reply easybuild/easyconfigs/l/libunwind/libunwind-1.3.1-GCCcore-8.3.0.eb]: fixed in d3722a1

File: easybuild/easyconfigs/l/libunwind/libunwind-1.3.1-GCCcore-8.3.0.eb (hunk 2 above)
Comment (furstj) [reply easybuild/easyconfigs/l/libunwind/libunwind-1.3.1-GCCcore-8.3.0.eb]: fixed in d3722a1

**Comment by Micket** [comment]
Going in, thanks @furstj!

**Pull request merged**